drivers/holykell_h5620.json

```diff
@@ -4,8 +4,12 @@
         "order": "lsr"
     },
     "fields": {
-        "liquid_level": {"datatype": "float", "description": "liquid level", "register": 0, "words": 2},
+        "level": {"datatype": "float", "description": "liquid level", "register": 0, "words": 2},
         "volume": {"datatype": "float", "description": "volume", "register": 2, "words": 2},
-        "quality": {"datatype": "float", "description": "quality", "register": 4, "words": 2}
+        "quality": {"datatype": "float", "description": "quality", "register": 4, "words": 2},
+        "alarm_state_1": {"datatype": "int32", "description": "1st alarm state", "register": 6, "words": 1},
```

Comment:
These would actually be type `uint16` - since they're only 1 word = 2 bytes = 16 bits. It doesn't actually matter if we set them as `int` vs `uint`, but since they're unsigned by design we may as well use `uint`. FYI the supported datatypes are here: https://github.com/ammpio/ammp-edge/blob/main/src/processor/process_reading.py#L82